### ISSUE TITLE
Add about dialogue with version Info

### ISSUE THREAD

**Issue by sirtet** [ewerybody/a2#265]
I wanted to see if my a2 was the latest version, so i checked the usual place, in the Menu: Help>About...
But there's no dialogue with version info, as i expected. It's just a Web-Link to the readme...
I think that's quite unusual and i suggest to implement the more standard behaviour of a dialogue window with version info in it.

**Comment by ewerybody** [comment]
Hey @sirtet! Thanks for the issue! Well, I did quite something on the backend of these things. If you open the **Console** tab in the main view after ui startup there should be something like:
```
1671479469.56 - INFO:a2ui:A2Window initialised! (0.391s)
 1671479469.56 - INFO:a2core:Checking for a2 updates ...
 1671479469.81 - INFO:a2core:Checking module package updates ...
 1671479470.25 - INFO:a2core:New "a2.modules" 0.6.1 module source package
```

There is a whole bunch of things I now check updates for. Of course this is the version of a2 itself and all loaded packages. (When developing from the code repo here there is also checks for latest Autohotkey, Qt for Python and the Python version itself.)
Now I "Just" need to make it properly visible for the user and offer some way to perform the update nicely.

**Comment by ewerybody** [comment]
I was thinking to implement some new UI for these kinds of things to have it non-blocking but very present to the user.
Would you personally like to have the flexibility of choosing to have updates downloaded automatically or on demand only?

**Comment by ewerybody** [comment]
status:
![image](https://user-images.githubusercontent.com/218956/213031469-071073af-501a-4673-92a3-eda613b0b715.png)

**Comment by ewerybody** [comment]
Of course this also needs to work properly in release as well. But looking good so far :)

**Comment by ewerybody** [comment]
implemented with [0.5.0](https://github.com/ewerybody/a2/releases/tag/0.5.0)
